src/test/kotlin/fi/metatavu/oioi/cm/test/functional/tests/ApplicationTestsIT.kt

```diff
@@ -100,28 +101,51 @@ class ApplicationTestsIT : AbstractFunctionalTest() {
             val customer = builder.admin().customers.create()
```

Comment:
Add test that tries to update application with invalid activeContentVersionResourceIds (non existing resource, not content type version, resource from another application)